package.json

```diff
@@ -25,6 +25,7 @@
     "draft-js-export-html": "^1.4.1",
     "draft-js-import-html": "^1.4.1",
     "draftjs-filters": "^2.3.0",
+    "polished": "^3.4.4",
```

Comment:
Should it be a `dep` or a `devDep`? @swissspidy?

Reply:
`dep` makes sense. We should actually move way more there from the dev deps... I'd need to go through the list.